lighthouse-core/report/html/renderer/dom.js

```diff
@@ -213,27 +213,42 @@ class DOM {
   /**
    * Guaranteed context.querySelector. Always returns an element or throws if
    * nothing matches query.
+   * @template {string} T
    * @param {string} query
    * @param {ParentNode} context
-   * @return {!HTMLElement}
+   * @param {T=} tagName
+   * @return {HTMLElementByTagName[T]}
    */
-  find(query, context) {
+  find(query, context, tagName) {
     /** @type {?HTMLElement} */
     const result = context.querySelector(query);
     if (result === null) {
       throw new Error(`query ${query} not found`);
     }
+    if (tagName && result.tagName.toLowerCase() !== tagName) {
```

Comment:
tagName always returns uppercase... so instead of tolowercaseing it, you should `tagName.toUpperCase()`